useful workflows/womtool-to-cbas.py

```diff
@@ -0,0 +1,113 @@
+'''
+A script for converting the output of womtool's `inputs` command to a CBAS input definitions file.
+
+This script was written (quickly) to accelerate the test and development of CBAS features,
+and should not be considered to be actively maintained. Use at your own risk.
+
```

Comment:
I think it would be great to add a comment on how to use this script as well as an example (like sample WomTool inputs and what it's CBAS input will look like through the script)

Reply:
done!